test/apps/collision/app.jsx

```diff
@@ -110,10 +111,10 @@ export default function App() {
         pointType: 'text',
         getText: f => f.properties.name,
         getTextColor: [0, 0, 0],
-        getTextSize: 12,
+        textSizeScale: 12,
         parameters: {depthTest: false},
 
-        extensions: [new CollisionFilterExtension(), new MaskExtension()],
+        extensions: [new MaskExtension()],
```

Comment:
@felixpalmer was this change intended?

Reply:
Yes a `PointLabelLayer` is now used we don't need the extension (it is already enabled)